ansible/roles/amq_streams_operator/tasks/main.yml

```diff
@@ -43,6 +43,19 @@
     query: >-
       [?starts_with(metadata.name, '{{ amq_streams_operator_deployment }}')].metadata.name|[0]
 
+# need to fix
```

Comment:
this was to avoid setting the env variables if it had already been set. Need to fix it